examples/fairmotion_interface.py

```diff
@@ -19,15 +19,13 @@
 #### Constants
 ROOT = 0
 FILE_SUFFIX = "_fm_data"
-METADATA_DEFAULT = {
-    "default": {
-        "urdf_path": "../habitat-sim/data/test_assets/urdf/amass_male.urdf",
-        "amass_path": "../fairmotion/amass_test_data/CMU/CMU/06/06_12_poses.npz",
-        "bm_path": "../fairmotion/amass_test_data/smplh/male/model.npz",
-        "rotation": mn.Quaternion.rotation(mn.Deg(-90), mn.Vector3.x_axis())
-        * mn.Quaternion.rotation(mn.Deg(90), mn.Vector3.z_axis()),
-        "translation": mn.Vector3([2.5, 0.0, 0.7]),
-    }
+METADATA_DEFAULT_WHEN_MISSING_FILE = {
+    "urdf_path": "../habitat-sim/data/test_assets/urdf/amass_male.urdf",
```

Comment:
If you are running it from the git root `../habitat-sim` is canceling each other out (all file paths are relative from the CWD which is where the python executable is run from.).